docs/EVENTS.md

```diff
@@ -215,6 +215,12 @@
 | customer_registration_is_allowed | 1.9.4.5 |
 | customer_session_init | 1.9.4.5 |
 | eav_collection_abstract_load_before | 1.9.4.5 |
+| email_queue_send_before | 19.4.18 |
```

Comment:
Don't forget to update numbers.

Reply:
I'll merge and update right after, just not to invalidate the reviews